cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -205,7 +205,7 @@ func (s *CMServer) Run(_ []string) error {
 	// file, and then overriding the Master flag, if non-empty.
 	kubeconfig, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
 		&clientcmd.ClientConfigLoadingRules{ExplicitPath: s.Kubeconfig},
-		&clientcmd.ConfigOverrides{ClusterInfo: clientcmdapi.Cluster{Server: s.Master}}).ClientConfig()
+		&clientcmd.ConfigOverrides{ClusterInfo: clientcmdapi.Cluster{Server: s.Master}}).ClientConfig("")
```

Comment:
This is just a temporary fix to make the compiler happy.
We should get config for all groups and pass a list of configs to client.New()